### DISCUSSION TITLE
Is it possible to exclude SystemD from mandatory dependencies?

### DISCUSSION THREAD

**Discussion by karabanov** [oerdnj/deb.sury.org#2240] [Q&A] [answered]
Hi

If you build a Docker image that includes PHP-FPM , then along with other dependencies, SystemD and its components are installed, but they are not used in any way in the future

Is it possible to exclude SystemD and its components from the mandatory dependencies so as not to add extra files to the Docker image?

**Comment by oerdnj** [comment]
Read the FAQ

**Reply by karabanov** [reply, marked as answer]
I read the FAQ. Got it, thanks.

It's a shame that creating temporary files requires a third-party component, even if it's SystemD, but oh well.
